code/espurna/config/hardware.h

```diff
@@ -4012,6 +4012,31 @@
     #define HLW8012_INTERRUPT_ON        FALLING
 
 
+// -----------------------------------------------------------------------------
+// MIOBULB001
+// -----------------------------------------------------------------------------
+
+#elif defined(MIOBULB001)
```

Comment:
```cpp
#elif defined(MUVIT_IO_MIOBULB001)
```
common way to determine the board is to `MANUFACTURER "_" DEVICE`

Reply:
defined MUVIT_OI_MIOBULB001